src/components/application_manager/test/commands/mobile/speaker_request_test.cc

```diff
@@ -284,11 +285,32 @@ TEST_F(SpeakRequestTest,
 
 TEST_F(SpeakRequestTest, OnEvent_TTS_OnResetTimeout_UpdateTimeout) {
   Event event(Event::EventID::TTS_OnResetTimeout);
-  (*msg_)[am::strings::params][am::strings::connection_key] = kKey;
+  (*msg_)[am::strings::params][am::strings::connection_key] = kConnectionKey;
   (*msg_)[am::strings::params][am::strings::correlation_id] = kAppId;
   CommandPtr command(CreateCommand<SpeakRequest>(msg_));
 
-  EXPECT_CALL(mock_app_manager_, updateRequestTimeout(kKey, kAppId, _));
+  EXPECT_CALL(mock_app_manager_,
+              updateRequestTimeout(kConnectionKey, kAppId, _));
+
+  command->on_event(event);
+}
+
+TEST_F(SpeakRequestTest, OnEvent_ApplicationIsNotRegistered) {
+  hmi_apis::Common_Result::eType hmi_result = hmi_apis::Common_Result::SUCCESS;
```

Comment:
@wolfylambova22, if it does not change during a test execution, please, make it `const`. Please check everywhere